internal/api/handlers.go

```diff
@@ -190,6 +191,7 @@ func GetRecommendationSet(c echo.Context) error {
 		longTermSection["variation"] = variationDummyObject
 
 		recommendationSlice["id"] = recommendationSet.ID
+		recommendationSlice["source_id"] = recommendationSet.Workload.Cluster.SourceId
```

Comment:
Do we need source_id in API responses?

Reply:
Not sure, I thought QE might need for verification so added it.

Reply:
Posted a question in the Arch doc for Chris regarding the same

Reply:
This is good to go, change approved on doc